### ISSUE TITLE
accessibility issue 14 (medium) - focus 'all fields' select, focus on carousel nav, constrast,focus on close button

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#259]
•	The focus is not visible on the the “All Fields” select input.
•	There is no focus indicator outline for the carousel’s previous and next buttons (only a change to a darker background color).
•	A focus outline should be implemented with sufficient color contrast (3:1 ratio) and at least a 2px width.
•	The focus outline does not have enough contrast in the footer.
•	There is no focus indicator on the close (X) button when the modal dialog is opened. (The modal dialog that appears when the “more” link is activated under the filter heading “creator”.)

**Comment by yulgit1** [comment]
addressed:
391f4eeb2d3d48d528232b9196746d171ee45d31
b9d8fc35b9850cd1ea503374acb5bca078afa245
dcf1bec4e0f911f51e25089a359563d963a7c9b5
